src/Codeception/Subscriber/Console.php

```diff
@@ -536,7 +536,9 @@ protected function detectWidth()
             }
         } elseif ($this->isWin() && (php_sapi_name() === "cli")) {
             exec('mode con', $output);
-            preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            if (isset($output[4])) {
+                preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            }
             if (!empty($matches[1])) {
```

Comment:
I thought about that too, but did not see a notice at runtime.  Turns out that `empty()` avoids it if the variable doesn't exist:  `empty() does not generate a warning if the variable does not exist.` (http://php.net/manual/en/function.empty.php)